lib/typescript/httpclient/payload/ExploreChannelRequestPayload.ts

```diff
@@ -1,4 +1,3 @@
 export interface ExploreChannelRequestPayload {
-  source: string;
```

Comment:
There's no generic explore endpoint anymore so since we're changing this we should rename this one